tasks/README.md

```diff
@@ -0,0 +1,95 @@
+# Tasks
+
+## Task object
+
+```
+{
+  "type":"Dev | Group",
+  "links": [
+    <link1>,
+    <link2>
+  ],
+  "endsOn":"<unix timestamp>",
+  "startedOn":"<unix timestamp>",
+  "status": "Active",
+  "ownerId":"<app owner user id>",
+  "percentCompleted":10,
+  "dependsOn": [
+    <task_id>,
+    <task_id>
+  ],
+  "participants": [list of users participating in this tasks. For 'group' type only],
```

Comment:
@sumitd94 , what is meant by the list of users? Will it be the userIds or usernames or the whole user object?

Reply:
Array iof user ids, fixed this.